Source/Core/Core/IOS/ES/ES.h

```diff
@@ -92,8 +78,8 @@ class ES final : public Device
 
     u16 gid = 0;
     u32 uid = 0;
+    // The same context is used for both title imports and exports.
     TitleImportContext title_import;
```

Comment:
Yeah, this occurred to me too, but likewise I couldn't think of anything much better. Is something like `TitleImportExportContent title_import_export` too verbose?